config/sidekiq.yml

```diff
@@ -5,6 +5,7 @@
   - [default, 2]
   - [claims, 2]
   - [stats_reports, 1]
+  - [convert_document, 1]
```

Comment:
**not a blocker** might be worth moving these to the top too so they [are always processed first](https://github.com/mperham/sidekiq/wiki/Advanced-Options#queues)